### DISCUSSION TITLE
Split DNS on Technitium

### DISCUSSION THREAD

**Discussion by dxfoxd** [TechnitiumSoftware/DnsServer#880] [Q&A]
I can find my way around with basic DNS settings, but the latest development in my home based 3CX server are more complicated. Updating my 3CX installation om premises to a newer version requires a split DNS option, which I obviously cannot set in my router (this device only allows to set 3 DNS IP's). While several explanations are given on https://www.3cx.com/docs/creating-fqdn-split-dns/, I want to use the T DNS server to do this as a new step moving forward. Is the T DNS server app capable to setup a split DNS and if yes, how can I configure this based on the explanation given on the aforementioned link? I did create a new zone but i cannot get the FQDN to resolve so I probably miss some configuration. Any help is appreciated.

Art.

**Comment by ShreyasZare** [comment]
Thanks for asking. If you just want a public domain name to resolve to a private IP address then you can just create a Conditional Forwarder zone for that domain name with "This Server" as the forwarder. Once the zone is created, just add the A record with the IP address you want to resolve.

If you are looking to have split horizon setup for a domain name you own and it is being hosted on Technitium DNS server then take a look at the discussions [here](https://github.com/TechnitiumSoftware/DnsServer/discussions/721) and [here](https://github.com/TechnitiumSoftware/DnsServer/discussions/815).

**Reply by dxfoxd** [reply]
Hi,

Thank you for your reply a few months ago. I now found the time to do another attempt to get it working, Unfortunately this does not give the result I want.

Lets say I have a domain abc.com  which is hosted externally. I created a n A record on their DNS server pointing to my static WAN 188.1.1.1 and a SRV record that routes traffic to a specific port 5001 for SIP traffic over UDP. The port forwarding on my router is sety to route incoming traffic to the local IP 192.168.10.68:5001 This works for clients outside my local LAN. However, I still have to do some research that routes all trafic to this port only based on the voip.abc.com (whithout providing the port).

For clients within my LAN IP range 192.168.10.*, I want all traffic to voip.abc.com to route it directly to local IP 192.168.10.68:5001.
Based on your first reply I have created a conditional forwarder, 
Name: @.abc.com
Use this-server enabled
and an A record for this purpose:
Name: @.abc.com
IPv4 Address: 192.168.10.68

This did not work, even if i use @.voip.abc.com I can still see the ping replies coming from the public IP address.

I also tried using an APP record (SplitHorizon.SimpleAddress) with the config
```
{
  "192.168.10.0/25": [
    "192.168.10.68"
  ],
  "0.0.0.0/0": [
    "https://voip.abc.com:5001"
  ]
}
```
But this also did not work.

Did I miss something here?

Thx for your help.

Art.

**Reply by ShreyasZare** [reply]
Thanks for the details. For testing, never using ping command. Always use the DNS Client tool that is available on the DNS admin panel. 

For this setup, just adding the A record in the forwarder zone will work. You do not need to use the APP record for Split Horizon app here. Also note that the APP record's json config will take only IP address so the URL in the config is not correct.

Setup the forwarder zone again with just the A record with your private IP and the FWD record that forwards to `this-server` and test with the DNS Client. Do share the output you see in the DNS Client.
